bitnami/common/templates/_secrets.tpl

```diff
@@ -55,3 +55,48 @@ Params:
 
 {{- printf "%s" $key -}}
 {{- end -}}
+
+{{/*
+Generate secret password or retrieve one if already created.
+
+Usage:
+{{ include "common.secrets.manage.password" (dict "secret" "secret-name" "key" "keyName" "customValues" (list .Values.path.to.the.password1 .Values.path.to.the.password2) "length" "password-length" "strong" false "context" $) }}
+
+Params:
+  - secret - String - Required - Name of the 'Secret' resource where the password is stored.
+  - key - String - Required - Name of the key in the secret.
+  - customValues - Optional - List of parameters that can be used by the user to define the password. Will pick first parameter with a defined value.
```

Comment:
Why not only `{given|custom|provided}Value` in singular and as `string`? Do you know any case where a password could be set via two different keys?

Reply:
AFAIK, only [bitnami/postgresql](https://github.com/bitnami/charts/blob/86b98162b81dccc144615f0c5bd5313e7e57c905/bitnami/postgresql/templates/_helpers.tpl#L48) and [bitnami/postgresql-ha](https://github.com/bitnami/charts/blob/86b98162b81dccc144615f0c5bd5313e7e57c905/bitnami/postgresql-ha/values.yaml#L5) have that option, but I included it nonetheless.

Reply:
do you mean the global one?

Reply:
Yes. Another option would be to implement a helper in those charts when `lookup` is added and avoid having `customValues ` as a list.